.github/workflows/pr-rebase-needed.yml

```
@@ -1,12 +1,16 @@
-name: "PR Needs Rebase"
+name: "PR conflicts"
 
 on:
   push:
   pull_request_target:
     types: [synchronize]
 
+concurrency:
+  group: pr-conflicts-${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
```

If you manually include a prefix, you shouldn't need to include the workflow name (`github.workflow`, see [github context](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context)).

```suggestion
concurrency:
  group: pr-rebase-needed-${{ github.event.pull_request.number }}
  cancel-in-progress: true
```

---

Right, I guess a more portable version would be 

```
${{ github.workflow }}-${{ github.event.pull_request.number }}
```